pkg/sctp/utils/utils.go

```diff
@@ -0,0 +1,202 @@
+// Copyright 2021-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"bytes"
+	"encoding/binary"
+	"net"
+	"sync"
+	"unsafe"
+
+	syscall "golang.org/x/sys/unix"
+)
+
+// BoolToInt ...
+func BoolToInt(b bool) int {
+	if b {
+		return 1
+	}
+	return 0
+}
+
+func checkEndian() binary.ByteOrder {
```

Comment:
This is a bad name for a function... It's not checking anything. It's getting the platform byte order.

Reply:
@kuujo what should I name it? what do you suggest?

Reply:
`getNativeByteOrder`